src/main/payment/main/G1HRPS/PaymentManager.java

```diff
@@ -184,7 +205,9 @@ public Payment GenerateAndPrintBill(String guest_id, int room_num, int days_of_s
 		for(RoomServiceOrder each_order : room_service_orders) {
 			price_of_order = each_order.CalTotalPrice();
 			System.out.println("--- Room Service Order [" + index + "]:");
-
+                        
+			// Prints all menu items in each order
+			
```

Comment:
Hm please add a // TODO: as a reminder that this needs be changed. then the below code can be removed